library/lib_num_to_formatted_str.ks

```diff
@@ -11,33 +11,40 @@ LOCAL lib_formatting_lex IS LEX().
 }
 
 LOCAL FUNCTION time_converter {
-  PARAMETER timeValue,places.
+  PARAMETER timeValue,  // the time in seconds to convert
+  places.               // how many time places (e.g HH:MM:SS has 3 places)
+
   LOCAL returnList IS LIST().
   LOCAL localTime IS timeValue.
-
   LOCAL place IS 1.
+
   FOR modValue IN lib_formatting_lex["timeModList"] {
-    LOCAL returnValue IS MOD(localTime,modValue).
+    LOCAL returnValue IS MOD(localTime, modValue).
     returnList:ADD(returnValue).
 
     SET localTime TO FLOOR(localTime / modValue).
     IF localTime = 0 { BREAK. }
     SET place TO place + 1.
     IF place = places { BREAK. }
-
   }
+
   IF localTime > 0 { returnList:ADD(localTime). }
   RETURN returnList.
 }
 
 lib_formatting_lex:ADD("leading0List",LIST(2,2,2,3,3)).
-LOCAL FUNCTION time_string {
-  PARAMETER timeSec, fixedPlaces, stringList, roundingList, tMinus.
-  LOCAL places IS stringList:LENGTH.
-  LOCAL timeList IS time_converter(ABS(timeSec),places).
-  LOCAL returnString IS "".
+FUNCTION time_string {
```

Comment:
changing the `time_string` function from local to global means that you need to document it in the documentation page.

Reply:
Agreed. reverted back to LOCAL FUNCTION.